qiime2/tests/data/metadata/empty-rows.tsv

```diff
@@ -0,0 +1,8 @@
+my-index	col1	col2	col3
+id1	1	a	foo
```

Comment:
Excel tends to add also spaces (not sure why or how) to the empty rows/cols, could you add these 2 to your test? Basically, add another "blank" column/row with empty spaces.